### DISCUSSION TITLE
PHP 8 support

### DISCUSSION THREAD

**Discussion by sercan-uzun** [jstolpe/instagram-graph-api-php-sdk#2] [Q&A] [answered]
Hi @jstolpe ,
Thanks for useful API. When I tried to install on my server, composer give me PHP version error below. I have PHP 8. Do you planning to compatible with version of the PHP 8?

`  Package jstolpe/instagram-graph-api-php-sdk has requirements incompatible with your PHP version, PHP extensions and
   Composer version:
    - jstolpe/instagram-graph-api-php-sdk v1.0.3 requires php ^5.6|^7.0 which does not match your installed version 8
  .1.10.`

**Comment by jstolpe** [comment, marked as answer]
I have updated the composer.json with php 8.

**Comment by darkwob** [comment]
Hi, @jstolpe

how can ı get instagram user id ? 

Is there a way to do this? I entered the first site that came up when I searched the internet as “ig user id” and gave me ig user id, I added it with code, I added the token in the graph explorer tool I received by logging in with my facebook account, which is my developer account, to the token variable, it works, but I get the following error, what is the problem?  

Unsupported get request. Object with ID '3141496438' does not exist, cannot be loaded due to missing permissions, or does not support this operation. Please read the Graph API documentation at https://developers.facebook.com/docs/graph-api
